### ISSUE TITLE
upgrade to intl 0.17

### ISSUE THREAD

**Issue by Miamoto-Musashi** [ilteoood/flutter_i18n#157]
else build brakes 

```bash
Because flutter_i18n >=0.21.0 depends on intl ^0.16.0 and every version of flutter_localizations from sdk depends on intl 0.17.0, flutter_i18n >=0.21.0 is incompatible with flutter_localizations from sdk.

So, because project depends on flutter_i18n ^0.21.1 which depends on flutter_localizations any from sdk, version solving failed.
```

**Comment by dukaric1991** [comment]
Same issue here, after upgrading flutter from 1.22.6 to 2.0.0

**Comment by ilteoood** [comment]
It should be fixed with 0.22.0
